EasyPost/Services/WebhookService.cs

```diff
@@ -30,6 +30,7 @@ internal WebhookService(EasyPostClient client) : base(client)
         [CrudOperations.Create]
         public async Task<Webhook> Create(Dictionary<string, object> parameters)
         {
+            // parameters = parameters.Wrap("webhook");// TODO: Update docs to remove wrapped "webhook" key
```

Comment:
Although this may work without, please include it. When we are passing params of an associated object, they should always be wrapped.

Reply:
Resolved